addons/addon-base-workflow/packages/base-workflow-core/lib/schema/update-wf-assignment.js

```diff
@@ -0,0 +1,45 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License").
+ *  You may not use this file except in compliance with the License.
+ *  A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+const { nameRegex } = require('@amzn/base-services/lib/helpers/constants');
+
+const schema = {
+  $schema: 'http://json-schema.org/draft-07/schema#',
+  type: 'object',
+  additionalProperties: false,
+  properties: {
+    id: {
+      type: 'string',
+      minLength: 1,
+      maxLength: 200,
+      pattern: nameRegex,
```

Comment:
This should probably be `idRegex`. the create schema used `idRegex`.
This needs to be tested. There is no test coverage for these files:
- workflow-assignment-service.js

Reply:
non-block: This needs to be tested. There is no test coverage for workflow-assignment-service.js

Reply:
added to CHAM-212 and added idRegex in schema